doc/tutorials/open_vswitch.rst

```diff
@@ -0,0 +1,155 @@
+supports the following features:
+
+- Visibility into inter-VM communication via NetFlow,
+  sFlow(R), IPFIX, SPAN, RSPAN, and GRE-tunneled mirrors
+- LACP (IEEE 802.1AX-2008)
+- Standard 802.1Q VLAN model with trunking
+- Multicast snooping
+- IETF Auto-Attach SPBM and rudimentary required LLDP support
+- BFD and 802.1ag link monitoring
+- STP (IEEE 802.1D-1998) and RSTP (IEEE 802.1D-2004)
+- Fine-grained QoS control
+- Support for HFSC qdisc
+- Per VM interface traffic policing
+- NIC bonding with source-MAC load balancing, active backup, and L4 hashing
+- OpenFlow protocol support (including many extensions for virtualization)
+- IPv6 support
+- Multiple tunneling protocols (GRE, VXLAN, STT, and Geneve, with IPsec support)
+- Remote configuration protocol with C and Python bindings
+- Kernel and user-space forwarding engine options
+- Multi-table forwarding pipeline with flow-caching engine
+- Forwarding layer abstraction to ease porting to new software and hardware platforms
+
+For more detailed OVS information, please refer to `<https://www.openvswitch.org/>`_
+
+.. _enable_ovs_in_ACRN:
+
+How to enable OVS in ACRN
+*************************
+The OVS service is integrated in the ClearLinux ``service-os`` bundle.
+After booting the ACRN Service OS, start the OVS service with the command::
+
+    systemctl start openvswitch
+
+The default ``acrn-br0`` bridge is created by the SOS ``systemd`` and
+supports the UOS network.
+
+.. figure:: images/default-acrn-network.png
+   :align: center
+
+   Default ACRN Network
+
+.. note::
+   If ``systemctl start openvswitch`` fails to start OVS service,
+   you may also need to add ``network-basic`` bundle for other network dependencies
+   and run the ``systemctl`` command again.
+
+How to use OVS bridge instead of ACRN bridge
+********************************************
+#. Disable acrn network configuration::
+
+   # cd /usr/lib/systemd/network/
+   # mv 50-acrn.network 50-acrn.network_bak
+
+#. Modify ``50-eth.network`` to enable DHCP on OVS bridge
+
+   .. code-block:: none
+
+      [Match]
+      Name=ovs-br0
+
+      [Network]
+      DHCP=ipv4
+
+#. Create OVS bridge::
+
+   # ovs-vsctl add-br ovs-br0
+
+#. Add ``ens4``, ``acrn_tap_LaaG`` into OVS bridge::
+
+   # ovs-vsctl add-port ovs-br0 ens4
+   # ovs-vsctl add-port ovs-br0 acrn_tap_LaaG
+
+#. Modify ``launch_uos.sh`` script to enable ``acrn_tap_LaaG`` device before launching the UOS:
+
+   .. code-block:: none
+
+      # sed -i "s/virtio-net,tap0/virtio-net,acrn_tap_LaaG/" /usr/share/acrn/samples/nuc/launch_uos.sh
+
+   .. note::
+      If you set up the UOS via `acrn_quick_setup.sh
+      <https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/getting-started/acrn_quick_setup.sh>`_,
+      then replace ``/usr/share/acrn/samples/nuc/launch_uos.sh`` with ``/root/launch_uos_<version>.sh``
+      in ``sed`` command above.
+
+#. The UOS and SOS network will work after rebooting the host via ``ovs-br0``
```

Comment:
Update this to use "User VM" and "Service VM" instead of "UOS" and "SOS" (throughout this doc).